massa-network/src/peer_info_database.rs

```diff
@@ -494,50 +460,35 @@ impl PeerInfoDatabase {
         if !ip.is_global() {
             return Err(NetworkError::InvalidIpError(*ip));
         }
-        let (available_bootstrap_conns, available_nonbootstrap_conns) =
-            self.get_available_out_connection_attempts();
+        let mut update_happened = false;
+        let peer_type = if let Some(p) = self.get_peer_type(ip) {
+            p
+        } else {
+            let mut p = PeerInfo::new(*ip, false);
+            p.active_out_connection_attempts += 1;
```

Comment:
this value is already incremented below, so the peer's active_out_connection_attempts will be 2 at the end of the function if the peer didn't exist before

Reply:
see comment above